tests/ImageSharp.Web.Benchmarks/Caching/StringJoinBenchmarks.cs

```diff
@@ -12,11 +12,13 @@ namespace SixLabors.ImageSharp.Web.Benchmarks.Caching
     public class StringJoinBenchmarks
     {
         private const string Key = "abcdefghijkl";
-        private const int CachedNameLength = 12;
+
+        [Params(0, 8, 12)]
+        public int CacheFolderDepth { get; set; }
```

Comment:
Because the generated file paths have different lengths based on the depth, I've tested this with 3 values:

|               Method | CacheFolderDepth |       Mean |     Error |    StdDev | Ratio | RatioSD |  Gen 0 | Allocated |
|--------------------- |----------------- |-----------:|----------:|----------:|------:|--------:|-------:|----------:|
|          String.Join |                0 |  12.047 ns | 0.2181 ns | 0.1933 ns |  1.00 |    0.00 |      - |         - |
| StringBuilder.Append |                0 |  30.396 ns | 0.3118 ns | 0.2764 ns |  2.52 |    0.05 | 0.0229 |     144 B |
|        String.Create |                0 |   2.432 ns | 0.0788 ns | 0.0698 ns |  0.20 |    0.01 | 0.0038 |      24 B |
|                      |                  |            |           |           |       |         |        |           |
|          String.Join |                8 | 200.307 ns | 4.0510 ns | 4.3345 ns |  1.00 |    0.00 | 0.0701 |     440 B |
| StringBuilder.Append |                8 |  61.438 ns | 1.2429 ns | 1.1626 ns |  0.31 |    0.01 | 0.0331 |     208 B |
|        String.Create |                8 |  26.131 ns | 0.2055 ns | 0.1923 ns |  0.13 |    0.00 | 0.0242 |     152 B |
|                      |                  |            |           |           |       |         |        |           |
|          String.Join |               12 | 253.192 ns | 4.2244 ns | 3.9515 ns |  1.00 |    0.00 | 0.0854 |     536 B |
| StringBuilder.Append |               12 |  90.561 ns | 0.9826 ns | 0.9191 ns |  0.36 |    0.01 | 0.0381 |     240 B |
|        String.Create |               12 |  37.491 ns | 0.4276 ns | 0.4000 ns |  0.15 |    0.00 | 0.0293 |     184 B |